lambdas/service/app.py

```diff
@@ -821,36 +825,81 @@ def get_order():
     return {'order': Plugin.load().service_config().order_config}
 
 
-@app.route('/manifest/files', methods=['GET'], cors=True)
-def start_manifest_generation():
-    """
-    Initiate and check status of a manifest generation job, returning a either a 301 or 302 response
-    redirecting to either the location of the manifest or a URL to re-check the status of the manifest job.
+def manifest_path_spec():
```

Comment:
Why is this a function and not a simple variable?

Reply:
As a variable the functionality of the `method_spec` parameter broke for the two manifest endpoints, causing the value from `/manifest/files` method_spec to be overwritten with the method_spec from `/fetch/manifest/files`.

Two possible solutions for this were to make the `manifest_path_spec` variable a function, or copy/deepcopy the variable when passed in as a `path_spec` parameter. The function approach seemed cleaner however I will change this to the copy approach instead.